go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/github/hub
```

Comment:
Do we need to add `/v2` here?

Reply:
Since this repository is a program, I don't think its needed.

Reply:
https://github.com/golang/go/wiki/Modules#semantic-import-versioning:

> If the module is version v2 or higher, the major version of the module _must_ be included in both the module path in the go.mod file (e.g., `module example.com/my/mod/v2`) and the package import path (e.g., `import "example.com/my/mod/v2/foo"`).

Reply:
Done.